cmd/libvirt-actuator/resources/user-data.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+cat <<HEREDOC > /root/user-data.sh
+#!/bin/bash
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+exclude=kube*
+EOF
+setenforce 0
+yum install docker -y
+systemctl start docker
```

Comment:
What about an additional `systemctl enable docker` should you ever reboot this instance?